doc/classes/Tween.xml

```diff
@@ -144,6 +144,12 @@
 				[b]Note:[/b] As it results from accumulating frame deltas, the time returned after the [Tween] has finished animating will be slightly greater than the actual [Tween] duration.
 			</description>
 		</method>
+		<method name="has_tweeners">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if [method tween_property], [method tween_interval], [method tween_callback] or [method tween_method] has been called.
```

Comment:
```suggestion
				Returns [code]true[/code] if [method tween_property], [method tween_interval], [method tween_callback], or [method tween_method] has been called.
```
Is this always the case? Is the vector never cleared even when it's done working?